ios-sdkTests/MapViewControllerTests.swift

```diff
@@ -224,6 +226,7 @@ class MapViewControllerTests: XCTestCase {
   }
   
   func testFindMeButtonInitialState() {
+    controller.viewDidLoad()
```

Comment:
Mmmmm we should open the scope of `setupFindMeButton()` to `Internal` so we can test it. That'll still accomplish keeping it private from implementing devs.